src/TreeDisplay/TreeTag.py

```diff
@@ -396,11 +396,11 @@ def tpRenderTABLE(self, id, root_url, url, state, substate, diff, data,
                 ptreeData['tree-item-expanded'] = 1
                 output('<a name="%s" href="%s?%stree-c=%s#%s">'
                        '<img src="%s/p_/mi" alt="-" border="0" /></a>' %
-                       (id, root_url, param, s, id, script))
+                       (id, root_url, param, s.decode('ascii'), id, script))
```

Comment:
On line 383 there is already a `decode('ascii')` taking place under Python 3:
```
if six.PY3:
    s = s.decode('ASCII')
 ```
Did this not work as expected?

Under Python 2 it shouldn't make a difference, as the comment on line 381 suggests `s` is bytes, which is the same as str and doesn't render with a b prefix.